infrastructure/Dockerfile

```diff
@@ -24,6 +24,11 @@ COPY target/logback.prod.xml /usr/share/corfu/conf/logback.prod.xml
 COPY target/corfu-compactor-config.yml /usr/share/corfu/conf/corfu-compactor-config.yml
 COPY target/compactor-logback.prod.xml /usr/share/corfu/conf/compactor-logback.prod.xml
 
+# Create directories and files required for running Corfu Server
+RUN mkdir -p /config
+RUN mkdir -p /var/log/corfu/
+RUN touch /var/log/corfu/corfu.jvm.gc.9000.log
```

Comment:
do we really need this one? The file will be created automatically

Reply:
Looks like we don't need this file `corfu.jvm.gc.9000.log`. Removed it now, thanks!